src/main/java/tr/edu/ogu/ceng/service/Exception/DataAccessException.java

```diff
@@ -0,0 +1,14 @@
+package tr.edu.ogu.ceng.service.Exception;
+
+@SuppressWarnings("serial")
+public class DataAccessException extends ServiceException {
+
+	public DataAccessException(String message) {
+		super(message);
+	}
```

Comment:
ServiceException sınıfında bu constructor varsa neden buraya da ekliyoruz?

Reply:
kendi istediğimiz bir mesaj olursa buraya parametre olarak verip superle ServiceException constructorına ordan da superle RuntimeException constructorına gönderebilmek için ekliyoruz, masterdaki base böyle oluşturulmuş.

Reply:
Peki şimdilik kalsın ilerleyen aşamalarda güncellenecek nasılsa,
 temiz kod için bu tarz sadece veri geçiren yöntemleri gerekmedikçe yazmayız.